From 393569aad7402a80c072ddf42e164de821635023 Mon Sep 17 00:00:00 2001
In-Reply-To: <xmqq5z4mjdbq.fsf@gitster.c.googlers.com>
References: <xmqq5z4mjdbq.fsf@gitster.c.googlers.com>
From: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
Date: Mon, 28 Dec 2020 18:14:11 +0100
Subject: [PATCH v2 0/2] CoC: update to 2.0 with less upstream divergence
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Here's an alternate version of this update which is hopefully easier
to read diff-wise, and produces a smaller delta with the upstream
document.

With this version the diff to upstream is ~30 lines, with Junio's it's
~60. Specifically for this one:

BEGIN DIFF

diff --git a/code_of_conduct.md b/CODE_OF_CONDUCT.md
index 4e2473b8399..65651beada7 100644
--- a/code_of_conduct.md
+++ b/CODE_OF_CONDUCT.md
@@ -1,5 +1,10 @@
+# Git Code of Conduct
 
-# Contributor Covenant Code of Conduct
+This code of conduct outlines our expectations for participants within
+the Git community, as well as steps for reporting unacceptable behavior.
+We are committed to providing a welcoming and inspiring community for
+all and expect our code of conduct to be honored. Anyone who violates
+this code of conduct may be banned from the community.
 
 ## Our Pledge
 
@@ -61,7 +66,13 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+git@sfconservancy.org, or individually:
+
+  - Ævar Arnfjörð Bjarmason <avarab@gmail.com>
+  - Christian Couder <christian.couder@gmail.com>
+  - Jeff King <peff@peff.net>
+  - Junio C Hamano <gitster@pobox.com>
+
 All complaints will be reviewed and investigated promptly and fairly.
 
 All community leaders are obligated to respect the privacy and security of the


END DIFF

And for Junio's:

BEGIN DIFF

diff --git a/code_of_conduct.md b/CODE_OF_CONDUCT.md
index 4e2473b8399..ac13a84d1a4 100644
--- a/code_of_conduct.md
+++ b/CODE_OF_CONDUCT.md
@@ -1,5 +1,10 @@
+# Git Code of Conduct
 
-# Contributor Covenant Code of Conduct
+This code of conduct outlines our expectations for participants within
+the Git community, as well as steps for reporting unacceptable behavior.
+We are committed to providing a welcoming and inspiring community for
+all and expect our code of conduct to be honored. Anyone who violates
+this code of conduct may be banned from the community.
 
 ## Our Pledge
 
@@ -60,8 +65,14 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+reported to the community leaders responsible for enforcement by
+email as a whole at git@sfconservancy.org, or individually:
+
+  - Ævar Arnfjörð Bjarmason <avarab@gmail.com>
+  - Christian Couder <christian.couder@gmail.com>
+  - Jeff King <peff@peff.net>
+  - Junio C Hamano <gitster@pobox.com>
+
 All complaints will be reviewed and investigated promptly and fairly.
 
 All community leaders are obligated to respect the privacy and security of the
@@ -115,20 +126,13 @@ the community.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+This Code of Conduct is adapted from the Contributor Covenant,
 version 2.0, available at
-[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
 
-Community Impact Guidelines were inspired by 
-[Mozilla's code of conduct enforcement ladder][Mozilla CoC].
+Community Impact Guidelines were inspired by
+Mozilla's code of conduct enforcement ladder.
 
 For answers to common questions about this code of conduct, see the FAQ at
-[https://www.contributor-covenant.org/faq][FAQ]. Translations are available 
-at [https://www.contributor-covenant.org/translations][translations].
-
-[homepage]: https://www.contributor-covenant.org
-[v2.0]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
-[Mozilla CoC]: https://github.com/mozilla/diversity
-[FAQ]: https://www.contributor-covenant.org/faq
-[translations]: https://www.contributor-covenant.org/translations
-
+https://www.contributor-covenant.org/faq. Translations are available
+at https://www.contributor-covenant.org/translations.

END DIFF

Ævar Arnfjörð Bjarmason (2):
  CoC: Update word-wrapping to match upstream
  CoC: update to version 2.0 + local changes

 CODE_OF_CONDUCT.md | 154 ++++++++++++++++++++++++++++++---------------
 1 file changed, 103 insertions(+), 51 deletions(-)

-- 
2.29.2.222.g5d2a92d10f8

